src/pages/lib/graphqlapi/upgrade-guide/q/platform/[platform].mdx

```diff
@@ -0,0 +1,138 @@
+export const meta = {
+  title: `Upgrade guide`,
+  description: `Learn how to upgrade from AWS AppSync SDK for JavaScript (Maintenance mode) to AWS Amplify.`
+};
+
+If you're using AWS AppSync SDK for JavaScript (Maintenance mode), we recommend you to upgrade to AWS Amplify library or use the [AWS AppSync Apollo links](https://github.com/awslabs/aws-mobile-appsync-sdk-js#aws-appsync-links-for-apollo-v3) with the Apollo GraphQL client.
+
+**What does Maintenance mean?**
+While in maintenance, the AWS AppSync SDK for JavaScript (V2) will continue to receive updates that ensure compatibility with backend services and security updates. No new features will be introduced in the AWS AppSync SDK for JavaScript (V2).
+
+**How long will Maintenance last?**
+AWS AppSync SDK for JavaScript (V2) will be in Maintenance for 12 months i.e until June 30, 2024, after which no new updates will be made. This may result in the APIs going out of sync with the services and result in breaking production apps.
```

Comment:
```suggestion
AWS AppSync SDK for JavaScript (V2) will be in Maintenance for 12 months (June 30, 2024), after which no new updates will be made. This may result in the APIs going out of sync with the services and may result in broken production apps.
```